app/src/main/res/xml/preferences.xml

```diff
@@ -141,6 +128,12 @@
     </PreferenceCategory>
 
     <PreferenceCategory android:title="@string/miscellaneous">
+        <Preference
+            android:key="zippath"
+            android:summary="@string/zip_summary"
+            android:title="@string/zip_create_folder">
+        </Preference>
```

Comment:
I thought you were saying to remove the setting of "folder to create zips", but you are removing the "folder to extract zips" setting instead.

Reply:
Corrected now.